### DISCUSSION TITLE
Supply chain emission factors for other economies?

### DISCUSSION THREAD

**Discussion by gsdipietro** [USEPA/supply-chain-factors#48] [Q&A] [answered]
Are you aware of any similarly derived supply chain emission factors that apply to European or Asian economies?

**Comment by gsdipietro** [comment]
Exiobase seems similar, and available in openLCA (not nearly as convenient as the supply chain emission factors).

**Comment by bl-young** [comment, marked as answer]
You could review [pymrio](https://pymrio.readthedocs.io/en/latest/) for some other international multi regional input-output models.

**Reply by gsdipietro** [reply]
Thank you.
